tests/test_core/test_component.py

```diff
@@ -35,19 +35,19 @@ def SimpleParamComponent(tag):
 async def test_component_with_var_args():
     @idom.component
     def ComponentWithVarArgsAndKwargs(*args, **kwargs):
-        return idom.html.div(*args, **kwargs)
+        return idom.html.div(kwargs, args)
 
-    assert ComponentWithVarArgsAndKwargs("hello", "world", my_attr=1).render() == {
+    assert ComponentWithVarArgsAndKwargs("hello", "world", myAttr=1).render() == {
```

Comment:
Is the camelCase intentional here?

Reply:
Nope, not sure why that was camel in the first place given this is mostly a reversion.